ros2pkg/ros2pkg/resource/ament_cmake/CMakeLists.txt.em

```diff
@@ -0,0 +1,82 @@
+cmake_minimum_required(VERSION 3.5)
+project(@(project_name))
+
+# Default to C99
+if(NOT CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 99)
+endif()
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+@[if dependencies]@
+@[for deb in dependencies]@
+find_package(@deb REQUIRED)
+@[end for]@
+
+@[if dependencies or create_cpp_library]@
+include_directories(
+@[if create_cpp_library]@
+  include
```

Comment:
it would be good if the script would also create the directory `include/<pkgname>` in that case (maybe even a header?). Otherwise using the package might result in warning since the include path doesn't exist.